runtime/vm/bindnatv.cpp

```diff
@@ -313,7 +313,7 @@ inlIntercept(J9VMThread *currentThread, J9Method *nativeMethod, const char *symb
 		if (0 == strcmp(symbolName, mappings[i].nativeName)) {
 			Trc_VM_INLIntercepted(currentThread, symbolName);
 			nativeMethod->methodRunAddress = J9_BCLOOP_ENCODE_SEND_TARGET(mappings[i].sendTargetNumber);
-			nativeMethod->extra = (void*)(UDATA)-1;
+			nativeMethod->extra = reinterpret_cast<void*>(J9_JIT_NEVER_TRANSLATE);
```

Comment:
I thought we were adding a constant for this.

Reply:
I looked at the uses of `J9_JIT_NEVER_TRANSLATE` and determined what this define really means is that we just never want to JIT compile the particular method. The uses of `J9_JIT_NEVER_TRANSLATE` do no intersect each other (including this use introduced here).

If I understand this code correctly it is also trying to ensure we never try to JIT compile this method. It seems appropriate to use `J9_JIT_NEVER_TRANSLATE` here.

Is my understanding of this use flawed?

Reply:
No, you're right. I'll withdraw my objection.